docs/reference/quality-model-documents/open-source-health.md

```diff
@@ -0,0 +1,198 @@
+to the [CVSS v3](https://nvd.nist.gov/vuln-metrics/cvss) qualitative severity 
+rating. CVSS is the industry de-facto standard for communicating the 
+characteristics and severity of vulnerabilities. 
+It is developed and maintained by the NIST.
+
+To score a 4 stars rating for this property, a producer should aim to
+have only low-risk vulnerabilities.
+
+## Dependency Freshness
+
+Maintaining the freshness of third-party dependencies not only helps
+product security but also ensures its future-proofness. Third-party open
+source dependency upkeep is a crucial activity for several reasons, and
+this job is oftentimes postponed as not deemed necessary, or for lack of
+resources. Consequences linked to lack of continuous dependency refresh
+might lead to risks not only related to the security of a software
+system (older dependencies are oftentimes vulnerable, for example), but
+also with maintenance (a minor release update in a third party
+dependency is less resource intensive than a major release update).
+
+For the evaluation of the freshness property, SIG determines how much a
+dependency is out-of-date by measuring how long a newer version has been
+available.
+
+To score a 4 stars rating for this property a producer should aim at
+having:
+
+- The percentage of dependencies that are out-of-date for more than 2
+years should not exceed 17%.
+- The percentage of dependencies that are out-of-date for more than 3,5
+years should not exceed 16%.
+- The percentage of dependencies that are out-of-date for more than 5
+years should not exceed 7%.
+
+## License use
+
+Incorporating third-party open source dependencies in software products
+automatically assumes accepting the license that comes with it. With
+incompatible or restrictive licenses, software products might expose
+themselves to legal risks. Such restrictive licenses may conflict with
+the end-user, with the distribution method, or the business model, and
+failure to comply with licensing terms can lead to legal repercussions.
+
+Note that the information provided regarding the licenses used by
+third-party open source dependencies and associated risks is for general
+informational and risk evaluation purposes only. SIG is not a legal
+entity, and as such the content does not represent legal advice. Doubts
+or concerns about the legal implications of specific licenses should be
+consulted with the legal department of one’s organization or from a
+qualified legal professional. The accuracy and applicability of legal
+information can vary and may require personalized legal guidance. SIG
+disclaim any liability for actions taken based on this information.
+
+For the evaluation of the license property, SIG measures the risks
+associated with a dependency license. A brief, non-exhaustive outline of
+licenses associated with open source dependencies, and risks evaluation,
+is in the table below:
+
+| Risk        | Non-exhaustive example of licenses                         |
```

Comment:
I think this table is good, but, if we have an overview (as an external URL or our own docs somewhere else) of the full lit of licenses and their risks maybe we could link to that?

Reply:
I think that sounds good, we can make a ticket for it as a follow-up! :)